demos/dialog/index.html

```diff
@@ -153,6 +155,30 @@
         Secondary
       </mwc-button>
     </mwc-dialog>
+
+    <mwc-button data-num="8" raised>Form Validation</mwc-button>
+    <mwc-dialog id="dialog8" heading="Form Validation">
+      <p>We can validate user input before closing our dialog.</p>
```

Comment:
I would avoid first-person verbiage like "we" when tech writing. Instead, this could be written as "The dialog can validate user input before closing."

Reply:
sounds good!

Reply:
done